### PR TITLE
fix: ticket sales end date error msg

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/6150

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Issue says it should contain ticket names as wel
l

**Comment by maze-runnar** [comment]
Screenshot from 2020-12-27 11-02-53

**Comment by iamareebjamal** [comment]
Wrap ticket name in quotes, and please run `pre-commit install` before committing

**Comment by maze-runnar** [comment]
> Wrap ticket name in quotes, and please run `pre-commit install` before committing

Command 'pre-commit' not found,

**Comment by maze-runnar** [comment]
Screenshot from 2020-12-27 16-34-05

**Comment by iamareebjamal** [comment]
Install virtual env of python 3.8

**Comment by maze-runnar** [comment]
Screenshot from 2020-12-27 17-14-58

**Comment by maze-runnar** [comment]
Screenshot from 2020-12-27 17-18-26

**Comment by iamareebjamal** [comment]
Have anydesk session on Tuesday meeting

File: app/api/tickets.py [hunk 1]
```diff
@@ -247,6 +247,12 @@ def before_update_object(self, ticket, data, view_kwargs):
                 "Can't delete a ticket that has sales",
             )
 
+        if data.get('event') and data.get('sales_ends_at') > event.ends_at:
+            raise UnprocessableEntityError(
+                {'sales_ends_at': '/data/attributes/sales-ends-at'},
+                "End of ticket sales date of "+data.get('name')+" cannot be after end of event date.",
```
Comment (iamareebjamal) [inline app/api/tickets.py:253]:
```suggestion
                f'End of ticket sales date of "{ data.get('name') }" cannot be after end of event date',
```

File: app/api/tickets.py [hunk 2]
```diff
@@ -79,7 +79,7 @@ def before_create_object(self, data, view_kwargs):
             if data.get('sales_ends_at') > event.ends_at:
                 raise UnprocessableEntityError(
                     {'sales_ends_at': '/data/attributes/sales-ends-at'},
-                    "Ticket end date cannot be greater than event end date",
+                    "End of ticket sales date of "+data.get('name')+" cannot be after end of event date.",
```
Comment (iamareebjamal) [inline app/api/tickets.py:82]:
```suggestion
                    f'End of ticket sales date of "{ data.get('name') }" cannot be after end of event date',
```

File: app/api/tickets.py [hunk 3]
```diff
@@ -247,6 +247,12 @@ def before_update_object(self, ticket, data, view_kwargs):
                 "Can't delete a ticket that has sales",
             )
 
+        if data.get('event') and data.get('sales_ends_at') > event.ends_at:
```
Comment (iamareebjamal) [inline app/api/tickets.py:250]: What if data does not have sales_ends_at

File: app/api/tickets.py (hunk 1 above)
Comment (maze-runnar) [reply app/api/tickets.py]: > f'End of ticket sales date of "{ data.get('name') }" cannot be after end of event date',

this is giving error.

**Comment by maze-runnar** [comment]
https://github.com/fossasia/open-event-server/pull/7540

**Pull request closed without merging**